src/test/java/org/jenkinsci/plugins/runselector/steps/RunSelectorStepTest.java

```diff
@@ -0,0 +1,101 @@
+package org.jenkinsci.plugins.runselector.steps;
+
+import hudson.model.Result;
+import hudson.model.queue.QueueTaskFuture;
+import org.apache.commons.lang.RandomStringUtils;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static java.lang.String.format;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Tests for the {@link RunSelectorStep}.
+ *
+ * @author Alexandru Somai
+ */
+public class RunSelectorStepTest {
```

Comment:
These are just some basic tests.
The tests for each selector will be in separate classes, e.g. SpecificRunSelectorTest. But that will be in different PRs.